code/modules/fabrication/designs/protolathe/designs_stock.dm

```diff
@@ -0,0 +1,81 @@
+/datum/fabricator_recipe/protolathe/stock
+	category = "Stock Parts"
+	path = /obj/item/stock_parts/capacitor
+
+/datum/fabricator_recipe/protolathe/stock/adv_capacitor
+	path = /obj/item/stock_parts/capacitor/adv
+
+/datum/fabricator_recipe/protolathe/stock/super_capacitor
+	path = /obj/item/stock_parts/capacitor/super
+
+/datum/fabricator_recipe/protolathe/stock/micro_mani
+	path = /obj/item/stock_parts/manipulator
+
+/datum/fabricator_recipe/protolathe/stock/nano_mani
+	path = /obj/item/stock_parts/manipulator/nano
+
+/datum/fabricator_recipe/protolathe/stock/pico_mani
+	path = /obj/item/stock_parts/manipulator/pico
+
+/datum/fabricator_recipe/protolathe/stock/basic_matter_bin
+	path = /obj/item/stock_parts/matter_bin
+
+/datum/fabricator_recipe/protolathe/stock/adv_matter_bin
+	path = /obj/item/stock_parts/matter_bin/adv
+
+/datum/fabricator_recipe/protolathe/stock/super_matter_bin
+	path = /obj/item/stock_parts/matter_bin/super
+
+/datum/fabricator_recipe/protolathe/stock/basic_micro_laser
+	path = /obj/item/stock_parts/micro_laser
+
+/datum/fabricator_recipe/protolathe/stock/high_micro_laser
+	path = /obj/item/stock_parts/micro_laser/high
+
+/datum/fabricator_recipe/protolathe/stock/ultra_micro_laser
+	path = /obj/item/stock_parts/micro_laser/ultra
+
+/datum/fabricator_recipe/protolathe/stock/basic_sensor
+	path = /obj/item/stock_parts/scanning_module
+
+/datum/fabricator_recipe/protolathe/stock/adv_sensor
+	path = /obj/item/stock_parts/scanning_module/adv
+
+/datum/fabricator_recipe/protolathe/stock/phasic_sensor
+	path = /obj/item/stock_parts/scanning_module/phasic
+
+/datum/fabricator_recipe/protolathe/stock/rped
```

Comment:
this prolly should go into tools with its bluespace bro

Reply:
Done.